templates/default/src/components/atoms/Country/Country.jsx

```diff
@@ -0,0 +1,48 @@
+// https://github.com/trevorblades/countries
+import React from 'react'
+import { useQuery } from '@apollo/react-hooks'
+import PropTypes from 'prop-types'
+import COUNTRY_QUERY from '../../../graphql/queries/country'
+
+const Country = ({code}) => {
+  const { loading, error, data } = useQuery(COUNTRY_QUERY, {
+    variables: { code },
+  });
+  const wrapperStyle = {
```

Comment:
Can you look at the internal version of this project and follow the pattern of using NextJS styled-jsx please